lib/ClangImporter/ImportDecl.cpp

```diff
@@ -2522,66 +2538,61 @@ namespace {
       Impl.SwiftContext.evaluator.cacheOutput(ExtendedTypeRequest{extension},
                                               enumDecl->getDeclaredType());
       Impl.SwiftContext.evaluator.cacheOutput(ExtendedNominalRequest{extension},
-                                              std::move(enumDecl));
+                                              enumDecl);
```

Comment:
Hmm, I guess moving a pointer is trivial, so it probably doesn't really matter. But I'm surprised this compiles, isn't this expecting an rvalue reference?

Reply:
Yes, it compiles fine :) The pointer is indeed trivially movable, so clang-tidy suggests removing `std::move` here.
We're using this `enumDecl` on L2593, and clang-tidy was also complaining about usage after move.

Reply:
Turns out you're right! It doesn't compile on Windows.